index.html

```diff
@@ -33,7 +33,8 @@
 
 <p>Beyond a browser, no desktop software will be necessary. We will be
 writing and trying out JavaScript programs directly inside of the
-browser.</p>
+browser. The <a href="https://developer.mozilla.org/en-US/docs/Web/API/Canvas_API/Tutorial" target=_blank>HTML5 canvas</a>
+element is used to draw graphics in these JavaScript programs.</p>
```

Comment:
While this is true it could be confusing as we are not using the canvas API directly but a thin custom API on top that is documented in the code examples.